Localization/frFR.lua

```diff
@@ -1,7 +1,7 @@
 local L = select(2, ...).L('frFR')
 
 -- Default string for keystone
-L['KEYSTONE'] = 'Keystone:'
+L['KEYSTONE'] = 'clé :'
```

Comment:
I have a problem with the links to the chats. the name is no longer linked to the object